### ISSUE TITLE
Bug Report: backend message is populating in toast error when id is undefined

### ISSUE THREAD

**Issue by ManashAnand** [PalisadoesFoundation/talawa-admin#1655]
**Describe the bug**
When id is undefined toast show a message coming from backend 
eg: cast to objectId failed for value {_id: undefined } ... 

**To Reproduce**

1. open talawa admin
2. on side nav go to block/unblock
3. if the blocked user is 0 
4. taost message with unexpected message shows up

**Expected behavior**
A clear and concise description of what happen should be there
from a user point of view
The proper message should be 
Id not found

**Actual behavior**
Message currently : cast to objectId failed for value {_id: undefined } ... 

**Screenshots**
![WhatsApp Image 2024-03-03 at 11 43 07 AM](https://github.com/PalisadoesFoundation/talawa-admin/assets/91553087/fa34e8fc-52fd-42f3-8d52-8621393d22b9)

![WhatsApp Image 2024-03-03 at 12 04 08 PM](https://github.com/PalisadoesFoundation/talawa-admin/assets/91553087/e4781e4c-9b61-46cd-819d-9eec22c639a0)

**Additional details**
Mostly I have worked on this issue 
and want to contribute 

Please assign me this issue as my starting point.

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

**Comment by ManashAnand** [comment]
Hey,
I would like to work on this issue for upcoming gsoc event 
Kindly assign this issue to me.

**Comment by Cioppolo14** [comment]
Unassigning due to no activity or open PR.

**Comment by GlenDsza** [comment]
@Cioppolo14 would like to work on this issue

**Comment by varshith257** [comment]
@Cioppolo14 Isn't it a duplicate of #1565

**Comment by pranshugupta54** [comment]
@ManashAnand, Are you still facing this issue? I'm not able to reproduce it

**Comment by pranshugupta54** [comment]
Can close this, not able to reproduce + this page will be removed and added under user's org page (Redesign pending)
